pages/Nvidia/_index.md

```diff
@@ -73,6 +73,11 @@ options nvidia NVreg_RegistryDwords="PowerMizerEnable=0x1; PerfLevelSrc=0x2222;
 
 Reboot your computer and it should be working.
 
+If it does not, try:
+- lowering your monitors' refresh rate, as this can stop the flickering altogether
+- using the [Nouveau driver](https://wiki.archlinux.org/title/Nouveau) as mentioned above
+- using an AMD GPU
```

Comment:
lol. Funny but doesn't belong here in all seriousness.

Reply:
Fair, but I do think the refresh rate workaround should be at least mentioned

Reply:
I am just complaining about point 3

Reply:
Removed point 3 and added driver version info as per other thread.